test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/PauseUnpauseTokenAccountPrecompileSuite.java

```diff
@@ -72,6 +64,9 @@ public class PauseUnpauseTokenAccountPrecompileSuite extends HapiApiSuite {
     private static final String PAUSE_NONFUNGIBLE_TXN = "pauseNonFungibleTxn";
     private static final String UNPAUSE_NONFUNGIBLE_TXN = "unpauseNonFungibleTxn";
 
+    private final AtomicReference<AccountID> accountID = new AtomicReference<>();
+    private final Object notAnAddress = new byte[0];
```

Comment:
nit: final var INVALID_ADDRESS = "0x0000000000000000000000000000000000123456";
This variable is used in some test cases, so perhaps keep up with that?

Reply:
Which test cases exactly? Didn't seem to find any with search everywhere and besides isn't it easier with an empty byte array?

Reply:
For example in 'BalanceOperationSuite'. There is no difference whether to be Object or String, but in my opinion it is clearer to explicitly declare invalid address - that is why I pointed it as a nit.